packages/client/tests/functional/issues/13097-group-by-enum/tests.ts

```diff
@@ -0,0 +1,69 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+/**
+ * Regression test for #13097
+ * groupBy on enum fields
+ */
+testMatrix.setupTestSuite(
+  ({ provider }) => {
+    beforeAll(async () => {
+      if (provider !== 'mysql') {
+        await prisma.resource.create({
```

Comment:
Wonder if it's worth adding 2 or 3 resource's here, here we only test against one resource. wdyt?

Reply:
I think we're good since the bug originated from the query plan.